crategen/cli.py

```diff
@@ -1,30 +1,39 @@
-import click
+"""CLI module for converting TES and WES data to WRROC."""
+
 import json
+
+import click
+
 from crategen.converter_manager import ConverterManager
 
+
 @click.command()
-@click.option('--input', prompt='Input file', help='Path to the input JSON file.')
-@click.option('--output', prompt='Output file', help='Path to the output JSON file.')
-@click.option('--conversion-type', prompt='Conversion type', type=click.Choice(['tes-to-wrroc', 'wes-to-wrroc']), help='Type of conversion to perform.')
+@click.option("--input", prompt="Input file", help="Path to the input JSON file.")
+@click.option("--output", prompt="Output file", help="Path to the output JSON file.")
+@click.option(
+    "--conversion-type",
+    prompt="Conversion type",
+    type=click.Choice(["tes-to-wrroc", "wes-to-wrroc"]),
+    help="Type of conversion to perform.",
+)
 def cli(input, output, conversion_type):
-    """
-    Command Line Interface for converting TES/WES to WRROC.
-    """
+    """Command Line Interface for converting TES/WES to WRROC."""
```

Comment:
Please include an "Args" section in all newly added or modified docstrings, where missing (and "Attributes", "Returns", "Raises", "Examples", where applicable). I will not comment on other instances of this.